Cartfile

```diff
@@ -1,6 +1,7 @@
-binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.3
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 5.8
```

Comment:
Zijn er nog toffe wijzigingen in de iOS SDK die ons kunnen helpen? Zoals Camera-related dingen? 🙏 (ben nog met het overzicht bezig, dus misschien goed om hier mee te nemen)

Reply:
Deze versie gebruikte wij al alleen de oude nav sdk natuurlijk niet dus voor ons geen wijzigingen